integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/query/BaseQueryIT.java

```diff
@@ -0,0 +1,122 @@
+package org.infinispan.test.integration.as.query;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.junit.InSequence;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * @since 9.0
```

Comment:
I would vote for removing this javadoc (the same applies for other tests) or putting something meaningful here.